.github/workflows/periodic-snapshot.yml

```diff
@@ -16,13 +16,14 @@ jobs:
         with:
           ssh-key: ${{secrets.ACTION_PUSH_KEY}}
       - run: |
-          date="$(date +%y.%m).0-dev"
-          gawk -i inplace -F: -v q=\" -v tag=$date '/^  "version": / { print $1 FS, q tag q ","; next} { print }' package.json
-          gawk -i inplace -F= -v q=\" -v tag=$date '/^__version__ =/ { print $1 FS, q tag q; next} { print }' redash/__init__.py
-          gawk -i inplace -F= -v q=\" -v tag=$date '/^version =/ { print $1 FS, q tag q; next} { print }' pyproject.toml
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
In my experience, `github-actions[bot]` using the following email address.
> 41898282+github-actions[bot]@users.noreply.github.com

For reference:
- https://github.com/sqlitebrowser/homebrew-sqlite3/pull/1#discussion_r1148304502
- https://github.com/orgs/community/discussions/26560

Reply:
Seems that it works without the long numerical id.

> - https://github.com/actions/checkout/discussions/479

Which one it better?

Reply:
made the change